### PR TITLE
Release 59.0.0

### PR DESCRIPTION
**Pull request by MajorLift**
This is the release candidate for v59.0.0.

- `@metamask/snaps-sdk@6.1.1` (patch)

### CODE REVIEW

**Review by Mrtenz — APPROVED** [review]

File: packages/snaps-sdk/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.1.1]
+### Fixed
+- Bump `@metamask/providers` from `17.0.0` to `^17.1.2` ([#2598](https://github.com/metamask/snaps/pull/2598))
+  - Resolves compatibility issue with tools that don't support package subpath exports (e.g. Browserify).
```
Comment (Mrtenz) [inline packages/snaps-sdk/CHANGELOG.md:12]:
```suggestion
```

**Review by Mrtenz — CHANGES_REQUESTED** [review]
Clicked approve too soon. See above.

File: packages/snaps-sdk/CHANGELOG.md (hunk 1 above)
Comment (MajorLift) [reply packages/snaps-sdk/CHANGELOG.md:12]: Got it. Should I recategorize to "changed" as well or leave as "fixed"?

Comment (Mrtenz) [reply packages/snaps-sdk/CHANGELOG.md]: I guess changed makes more sense.

File: packages/snaps-sdk/CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.1.1]
+### Fixed
```
Comment (MajorLift) [inline packages/snaps-sdk/CHANGELOG.md:10]:
```suggestion
### Changed
```

**Review by Mrtenz — APPROVED** [review]

**Pull request merged**